### DISCUSSION TITLE
Tapparelle con motori Somfy

### DISCUSSION THREAD

**Discussion by roby1961** [indomus/forum#831] [GENERALE - Discussioni generiche]
Buongiorno a tutti, scusate ma sono una New entry.. qualcuno di voi ha mai configurato in Home Assistant le tapparelle Somfy?
ho difficoltà nella configurazione... grazie

**Comment by Nik71git** [comment]
io ho le tende da sole Somfy... con cosa le usi? Tahoma?

**Comment by roby1961** [comment]
Le mie sono tapparelle e ho solo un pulsante locale che vorrei remotizzare
con Home Assistant... sai come si fa?
Grazie

Il giorno gio 7 ott 2021 alle ore 17:58 Nik71git ***@***.***>
ha scritto:

**Comment by roby1961** [comment]


**Reply by Nik71git** [reply]
Per integrare questo ti serve o il gateway tahoma oppure quello che ti avevo indicato io collegato ad un device a contatti puliti. In pratica integri quest'ultimo (come ho fatto a casa mia). Nel web trovi anche dei trasmettitori RTS che puoi collegare ad device come scritto sopra ma sinceramente non so la bontà di questi prodotti in quanto RTS è un protocollo propietario, criptato e Rolling code....

**Comment by roby1961** [comment]
grazie delle informazioni

Il giorno mar 12 ott 2021 alle ore 15:03 Nik71git ***@***.***>
ha scritto:
